hack/test/unit

```diff
@@ -18,43 +18,51 @@ TESTDIRS="${TESTDIRS:-./...}"
 
 mkdir -p bundles
 
-cd api
-api_pkg_list=$(go list $TESTDIRS)
+case "$TESTDIRS" in
+"./api"* | "./...")
+	api_pkg_list=$(go -C ./api list ".${TESTDIRS#./api}")
+	;;
+esac
```

Comment:
Tried to keep it POSIX(ish) here with a `case` to handle the wildcard, but as the script runs as `bash`, maybe there's no need.